### PR TITLE
selecting user in pal shows entities lastEditedBy user this session

### PR DESCRIPTION
**Pull request by howard-stearns**

### CODE REVIEW

**Comment by howard-stearns** [comment]
note to reviewers: all the one-line changes are to make the file pass lint, from previous work.

**Comment by howard-stearns** [comment]
test plan: You will need to users.
- Run scripts/system/pal.js from this PR. Select the other user (either from the People Action List, or by clicking the green dot on the other user in the scene). 
- The Entities that were last edited by the selected user IN THIS SESSION will have a red box around them.
- Toggling the selection off (e.g., by clicking a different user or by clicking the red dot on the user in the scene), will hide the red boxes.

Note: Some scenes may have entities with entity scripts that do a little editing by each user. Such entities will be highlighted if the selected user was the last to edit them at the moment you select them.

**Comment by hifi-gustavo** [comment]
Build console output for commit: c79c489002139c5a24e2674f518a29968f30d97b available at: https://gist.github.com/3e9c4c2c3a4b47c202035b420a7b72c3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9298-c79c489.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9298-c79c489.dmg

File: scripts/system/pal.js [hunk 1]
```diff
@@ -73,6 +73,45 @@ ExtendedOverlay.applyPickRay = function (pickRay, cb) { // cb(overlay) on the on
...
+// Similar, for entities
+//
+function HighlightedEntity(id, entityProperties) {
+    this.id = id;
+    this.overlay = Overlays.addOverlay('cube', {
+        position: entityProperties.position,
+        rotation: entityProperties.rotation,
+        dimensions: entityProperties.dimensions,
+        solid: false,
+        color: {
```
Comment (zfox23) [inline scripts/system/pal.js]: Does @AlanZimmerman agree with the color red for this cube overlay?

**Comment by zfox23** [comment]
[select shows entities lastEditedBy selected user this session](https://trello.com/c/aaivSWsO/33-matching-lasteditedby-to-selection)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 97780ac97e4f0b62da024943538a501f3a4541ce available at: https://gist.github.com/3e9c4c2c3a4b47c202035b420a7b72c3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9298-97780ac.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9298-97780ac.dmg

**Pull request merged**